docs/reference/ingest/search-inference-processing.asciidoc

```diff
@@ -102,10 +102,13 @@ Here, you'll be able to:
 1. Choose a name for your pipeline.
   - This name will need to be unique across the whole deployment.
   If you want this pipeline to be index-specific, we recommend including the name of your index in the pipeline name.
+  - If you do not set the pipeline name, a default unique name will be provided upon selecting an ML trained model.
```

Comment:
> an ML trained model.

I'm not sure about grammar here whether this should be `a` or `an`? You could consider changing it to `a trained model` to remove the grammar ambiguity.